### ISSUE TITLE
Publish Dataset: Cannot publish datasets created with handles, error indicates provider not supported.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#3509]
See RT 244247. Our dataverse instance has a number of unpublished datasets created using handles as the persistent identifier, before we moved to using EZID/DOI. Users cannot publish these datasets -they receive an error: 

This dataset may not be published because its DOI provider is not supported. Please contact Dataverse Support for assistance.

We need to be able to support this while being configured to use DOI for ongoing dataset creation.

**Comment by kcondon** [comment]
Turns out this is a known issue. We had migrated draft datasets with handles and did not set the new globalidcreatetime field in the dataset table. Our UI now uses this as a flag to indicate the persistent id was registered and without it, it won't publish. So the workaround is to first verify the handle was registered by checking if it resolves. Next set globalidcreatetime=createdate for that dataset from the dvobject table. Now the dataset can be published through the UI.

We need to make sure all draft datasets with handles are first registered, then have globalidcreatetime values.
